Add verify_image_cache fixture from PyVista (#3)
pyvista/pytest-pyvista: README.rst

Usage
-----

> Once installed, you only need to have a `pl.show()` in your test, the plugin will automatically manage the cache generation if it does not exists,

Reviewer: ```suggestion
Once installed, you only need to use the command `pl.show()` in your test and the plugin will automatically manage the cache generation if it does not exist,
```

These are the flags you can use when calling ``pytest`` in the command line:

- ``--reset_image_cache`` creates a new image for each test in
``tests/plotting/test_plotting.py`` and is not recommended except for
testing or for potentially a major or minor release. 

- You can use ``--ignore_image_cache`` if you are running on Linux and want to
temporarily ignore regression testing. Realize that regression testing
will still occur on our CI testing.

- When using ``--fail_extra_image_cache`` if there is an extra image in the cache, it will report as an error.

Test specific flags
-------------------
These are attributes of `verify_image_cache`. You can set them as `True` if needed in the beginning of your test function.

- high_variance_tests:  If necessary, the threshold for determining if a test will pass or not is 
incremented to another predetermined threshold. This is currently done due to the use of an unstable 
version of VTK, in stable versions this shouldn't be necessary.

- windows_skip_image_cache: For test where the plotting in Windows is different from MacOS/Linux.

- macos_skip_image_cache: For test where the plotting in MacOS is different from Windows/Linux.

> - skip: If you have a test that plots a figure, but you don't want to compare it's output against the cache,

Reviewer: ```suggestion
- skip: If you have a test that plots a figure, but you don't want to compare its output against the cache,
```
